configure.ac

```diff
@@ -32,6 +32,13 @@ AS_IF([test "x$enable_logs" != "xno"], [
 	AC_DEFINE([OPENDHT_LOG], [false], [Define if DHT logs are enabled])
 ])
 
+dnl Check for indexation
+AC_ARG_ENABLE([logs], [AS_HELP_STRING([--enable-indexation], [Disble DHT logs])])
```

Comment:
This line is incorrect.
By the way, there is some `Disble`

Reply:
done